<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NBI1 LED Control</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>70</y>
      <width>130</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 20pt &quot;Sans Serif&quot;;</string>
    </property>
    <property name="text">
     <string>LED #1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>120</y>
      <width>130</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 20pt &quot;Sans Serif&quot;;</string>
    </property>
    <property name="text">
     <string>LED #2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>170</y>
      <width>130</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 20pt &quot;Sans Serif&quot;;</string>
    </property>
    <property name="text">
     <string>LED #3</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>220</y>
      <width>130</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 20pt &quot;Sans Serif&quot;;</string>
    </property>
    <property name="text">
     <string>LED #4</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>270</y>
      <width>130</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 20pt &quot;Sans Serif&quot;;</string>
    </property>
    <property name="text">
     <string>LED #5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>70</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ON</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>70</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>OFF</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>120</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ON</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>120</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>OFF</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>170</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ON</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>170</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>OFF</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>220</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ON</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>220</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>OFF</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>270</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>ON</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_10">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>270</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>OFF</string>
    </property>
   </widget>
   <widget class="CALabel" name="caLabel">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>70</y>
      <width>190</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="pvname">
     <string>NB1_VAC_FG4_AI</string>
    </property>
    <property name="dmode" stdset="0">
     <enum>CALabel::Extension</enum>
    </property>
   </widget>
   <widget class="CALabel" name="caLabel_2">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>120</y>
      <width>190</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="pvname">
     <string>NB1_VAC_FG7_AI</string>
    </property>
    <property name="dmode" stdset="0">
     <enum>CALabel::Extension</enum>
    </property>
   </widget>
   <widget class="CALabel" name="caLabel_3">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>170</y>
      <width>190</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="pvname">
     <string>NB1_VAC_FG31_AI</string>
    </property>
    <property name="dmode" stdset="0">
     <enum>CALabel::Extension</enum>
    </property>
   </widget>
   <widget class="CALabel" name="caLabel_4">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>270</y>
      <width>190</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="pvname">
     <string>NB1_CRYO13_RIGHT_BOTTOM</string>
    </property>
    <property name="dmode" stdset="0">
     <enum>CALabel::Int</enum>
    </property>
   </widget>
   <widget class="CALabel" name="caLabel_5">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>320</y>
      <width>190</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="pvname">
     <string>NB1_CRB_TTC_R02</string>
    </property>
    <property name="dmode" stdset="0">
     <enum>CALabel::Int</enum>
    </property>
   </widget>
   <widget class="StaticGraphic" name="stLedSt1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="fillMode">
     <enum>StaticGraphic::Solid</enum>
    </property>
   </widget>
   <widget class="StaticGraphic" name="stLedSt2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="fillMode">
     <enum>StaticGraphic::Solid</enum>
    </property>
   </widget>
   <widget class="StaticGraphic" name="stLedSt3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="fillMode">
     <enum>StaticGraphic::Solid</enum>
    </property>
   </widget>
   <widget class="StaticGraphic" name="stLedSt4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="fillMode">
     <enum>StaticGraphic::Solid</enum>
    </property>
   </widget>
   <widget class="StaticGraphic" name="stLedSt5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>280</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="fillMode">
     <enum>StaticGraphic::Solid</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_11">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>130</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>START</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_12">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>10</y>
      <width>130</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>STOP</string>
    </property>
   </widget>
   <widget class="CALabel" name="caLabel_6">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>220</y>
      <width>190</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="pvname">
     <string>NB1_VAC_FG32_AI</string>
    </property>
    <property name="dmode" stdset="0">
     <enum>CALabel::Extension</enum>
    </property>
   </widget>
   <widget class="CADisplayer" name="caDisplay">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>320</y>
      <width>110</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="nuformat" stdset="0">
     <enum>CADisplayer::Pressure</enum>
    </property>
    <property name="pvname">
     <string>NB1_VAC_FG4_AI</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CADisplayer</class>
   <extends>QWidget</extends>
   <header>caDisplayer.h</header>
  </customwidget>
  <customwidget>
   <class>CALabel</class>
   <extends>QLabel</extends>
   <header>caLabel.h</header>
  </customwidget>
  <customwidget>
   <class>StaticGraphic</class>
   <extends>QWidget</extends>
   <header>staticGraphic.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
